src/cljs/main/broadfcui/common/notifications.cljs

```diff
@@ -269,54 +270,57 @@
+                         :border-bottom "2px solid rgb(176, 210, 57)"
+                         :box-shadow "rgba(0, 0, 0, 0.12) 0px 3px 2px 0px"
+                         :background "81px url('assets/header-left-hexes.svg') no-repeat,
                                        right url('assets/header-right-hexes.svg') no-repeat, rgb(116, 174, 67)"}}
-                [:img {:src "assets/terra-logo.svg"
-                      :style {:height "56px"
-                              :width "60px"
-                              :float "left"
-                              :margin "4px 10px"}}]
-                [:span {:style {:color "white"
-                                :height "66px"
-                                :display "inline-block"
-                                :vertical-align "middle"
-                                :margin-left "112px"
-                                :text-align "center"
-                                :font-weight "500"}}
-                  "On May 1st FireCloud will get a new look as it becomes "
-                  [:a {:href "https://software.broadinstitute.org/firecloud/blog?id=23627"
-                       :target "_blank"
-                       :style {:color "white"}}
-                    "powered by Terra."]
-                  " Please click "
-                  [:a {:href "http://app.terra.bio/"
-                       :target "_blank"
-                       :style {:color "white"}}
-                    "here"]
-                  " to test-drive the new experience."]]
-                  [:div {:style {:alignSelf "center"
-                                 :padding "1rem"
-                                 :position "absolute"
-                                 :top "0px"
-                                 :right "0px"}}
-                    [FoundationTooltip
-                      {:tooltip "Hide for now"
-                       :style {:borderBottom "none"}
-                               :position "left"
-                               :data-hover-delay 0
-                               :text [:button {:className "button-reset" :onClick #(swap! state assoc :dismissed? true)
-                                      :style {:display "block" :fontSize "1.5rem"
-                                              :color "white" :cursor "pointer"}}
-                              (icons/render-icon {} :close)]}]]])))})
+           [:img {:src "assets/terra-logo.svg"
+                  :style {:height "56px"
+                          :width "60px"
+                          :float "left"
+                          :margin "4px 10px"}}]
+           [:span {:style {:color "white"
+                           :height "66px"
+                           :display "inline-block"
+                           :vertical-align "middle"
+                           :margin-left "112px"
+                           :text-align "center"
+                           :font-weight "500"}}
+            (if (common/has-terra-return?)
```

Comment:
to satisfy the "Banner should be displayed when we release the redirect (not before)" part of the requirement, you could make this:
```
(if (and (common/has-terra-return?) (config/terra-redirects-enabled))
```